packages/components/src/index.js

```diff
@@ -79,8 +79,8 @@ export { default as RangeControl } from './range-control';
 export { default as ResizableBox } from './resizable-box';
 export { default as ResponsiveWrapper } from './responsive-wrapper';
 export {
-	default as __experimentalRovingTabIndex,
-	RovingTabIndexItem as __experimentalRovingTabIndexItem,
+	default as __unstableRovingTabIndex,
+	RovingTabIndexItem as __unstableRovingTabIndexItem,
 } from './roving-tab-index';
```

Comment:
Another option could be to remove the export entirely. The components are only used internally in the components package. They could be moved to the tree-grid folder until they're replaced.

Reply:
It sounds like a good idea to move it to `tree-grid` as a subfolder 👍 

I don't think it's used outside of `@wordpress/components` so it could be also removed from the list of exports here.